x/fees/keeper/msg_server.go

```diff
@@ -19,65 +20,95 @@ func (k Keeper) RegisterFee(
 	msg *types.MsgRegisterFee,
 ) (*types.MsgRegisterFeeResponse, error) {
 	ctx := sdk.UnwrapSDKContext(goCtx)
-	if !k.isEnabled(ctx) {
-		return nil, sdkerrors.Wrapf(types.ErrInternalFee, "fees module is not enabled")
+
+	params := k.GetParams(ctx)
+	if !params.EnableFees {
+		return nil, types.ErrFeesDisabled
 	}
 
 	contract := common.HexToAddress(msg.ContractAddress)
+
 	if k.IsFeeRegistered(ctx, contract) {
-		return nil, sdkerrors.Wrapf(sdkerrors.ErrInvalidRequest, "contract is already registered %s", contract)
+		return nil, sdkerrors.Wrapf(
+			types.ErrFeesAlreadyRegistered,
+			"contract is already registered %s", contract,
+		)
 	}
 
 	deployer := sdk.MustAccAddressFromBech32(msg.DeployerAddress)
-	deployerAccount := k.evmKeeper.GetAccountWithoutBalance(ctx, common.BytesToAddress(deployer.Bytes()))
-	if deployerAccount == nil {
-		return nil, sdkerrors.Wrapf(sdkerrors.ErrInvalidRequest, "deployer account not found %s", msg.DeployerAddress)
+	deployerAcc := k.evmKeeper.GetAccountWithoutBalance(ctx, common.BytesToAddress(deployer))
```

Comment:
IMO, it would be beneficial for the code readability to have a consistent style for variable names, either 
abbreviated or written in full, rather than a mixture of `DeployerAddress` and `DeployerAddr` or `deployerAccount` and `deployerAcc`.